### PR TITLE
update old sponsors

### PR DESCRIPTION
**Pull request by hzoo**
> accidentally did a push (thought I was on a branch so reverted it)

Just updates so it doesn't have our older sponsors since we haven't updated this in a long time. Will make a change after this to actually calculate amounts since not everyone does the monthly amount (either one offs, yearly, periodic donations).

Would also be good to show previous sponsors anyway in a different way (either below or a different view as appreciation), what webpack has is pretty nice.

preview
image

### CODE REVIEW

File: website/pages/en/index.js [hunk 1]
```diff
@@ -99,6 +99,9 @@ const SponsorTier = props => {
   );
   return (
     <div>
+      <h3>{props.title}</h3>
+      <div>are currently pledging {props.amount} per month</div>
```
Comment (hzoo) [inline website/pages/en/index.js:103]: adding a short description

File: website/pages/en/index.js [hunk 2]
```diff
@@ -189,26 +190,14 @@ const Hero = ({ language }) => (
...
+          <a href="https://github.com/babel/babel/releases/tag/v7.13.0">
+            changelog
+          </a>{" "}
+          for more details!
         </span>
       </div>
 
       <MiniRepl language={language} />
-
-      <h3>Special Sponsor</h3>
```
Comment (hzoo) [inline website/pages/en/index.js:198]: This is gone (could do like a this could be you but don't need to)

File: website/static/css/index.css [hunk 3]
```diff
@@ -141,51 +141,6 @@
   }
 }
 
-.sponsors-work .cards {
```
Comment (hzoo) [inline website/static/css/index.css:144]: unused styles

File: website/siteConfig.js [hunk 4]
```diff
@@ -35,7 +35,7 @@ const users = loadYaml("./data/users.yml").map(user => ({
   image: `/img/users/${user.logo}`,
 }));
 
-const sponsorsManual = loadYaml("./data/sponsors.yml").map(sponsor => ({
+const sponsorsManual = (loadYaml("./data/sponsors.yml") || []).map(sponsor => ({
```
Comment (nicolo-ribaudo) [inline website/siteConfig.js:38]: Do we still need this `sponsorsManual` logic?

Comment (nicolo-ribaudo) [reply website/siteConfig.js:38]: Well maybe we might need it again in the future

**Review by nicolo-ribaudo — APPROVED** [review]
Thanks!

File: website/siteConfig.js (hunk 4 above)
Comment (hzoo) [reply website/siteConfig.js:38]: Yeah the only reason why I kept!

**Pull request merged**